flowvision/layers/attention/se.py

```diff
@@ -3,6 +3,19 @@
 
 
 class SEModule(nn.Module):
+    """
```

Comment:
```suggestion
    """SEModule(channels, reduction, rd_channels, ...)
```

如果 docstring 第一行写的是 函数 的 signature，那么最后编译成 html 时，会“重载” sphinx 自动提取的 signature，就不会有截图里的那种怪异的类类型了。 上面的参数我是胡乱写的，你可以自己尝试确认下吧。

如果signature写在 类的 docstring 开头不起作用的话，就试着写在 `__init__` 的开头看看。

oneflow 里有类似的例子，不过我现在还来不及找，只是路过回复下。如果上面方法搞不定，我们在随时讨论下。

> 这个应该是docstring的问题，因为这个默认传参是一个class，所以我也不确定这儿的显示，我感觉这是正常的？

Reply:
@Aries-chen 世耿这边可以尝试一下～